network.go

```diff
@@ -53,96 +55,140 @@ type NetworkServer struct {
 }
 
 type NetworkCreateRequest struct {
-	Name         string        `json:"name"`
-	Labels       []interface{} `json:"labels,omitempty"`
-	LocationUuid string        `json:"location_uuid"`
-	L2Security   bool          `json:"l2security,omitempty"`
+	Name         string   `json:"name"`
+	Labels       []string `json:"labels,omitempty"`
```

Comment:
I'm afraid this might break the compatibility with Terraform.

Reply:
Indeed, this will need to be updated on the terraform side of things.
However since our tf provider has everything vendor'ed and checked in as well as the go module have a static reference (https://github.com/gridscale/terraform-provider-gridscale/blob/27f919b59d6183185ecc4597475a8f042a5d9fad/go.mod#L4) - this should be OK to go in, nevertheless.
@itakouna ?

Reply:
yes, we can proceed with the changes. thanks @fkr